src/app/containers/Brand/index.jsx

```diff
@@ -3,9 +3,23 @@ import Brand from '@bbc/psammead-brand';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 const BrandContainer = () => {
-  const { brandName, brandSVG } = useContext(ServiceContext);
+  const { brandName, brandSVG, frontpageUrl } = useContext(ServiceContext);
```

Comment:
Just a thought, do we need to define a default `brandSVG` here:

https://github.com/bbc/simorgh/blob/f745b0082b783ebd27fc2eee57bf6ce6138d6ffe/src/app/lib/config/services/index.js#L13-L22

as we have this that sets the default service to `default`:

https://github.com/bbc/simorgh/blob/f745b0082b783ebd27fc2eee57bf6ce6138d6ffe/src/app/contexts/ServiceContext/index.jsx#L19-L21

Reply:
I realise this isn't a change you've explicitly made, however it now fails more abruptly now when you don't filter out the `default` story. 

(also I don't see the point in having a default value if we're just going to filter it out 🤷‍♂)

Currently, `latest` throws a failed prop type, however still renders the component, without the SVG.
This branch would now throw a `TypeError` which causes the entire component to fail and not render.

I think this should be addressed before merging

Reply:
Handled failure in absence of brandSVG by adding a check for brandSVG. 

Regarding if a default brandSVG should be defined - it's been decided it's not necessary, because (with the added check) the component works without an SVG, and the prop errors are currently not an issue